src/orcania.h

```diff
@@ -10,6 +10,10 @@
 #ifndef __ORCANIA_H__
 #define __ORCANIA_H__
 
+#ifdef _WIN32
+#define _WIN
```

Comment:
I'm asking a question about the name, it's a Windows port but specific to MinGW-w64, so wouldn't it be better to name the define related to MinGW-w64 rather than Windows, if in a near future someone is able to build Ulfius on VCC with native Windows API for example?

```C
#define _MINGW
```

What do you think?

Reply:
I'm not sure if MinGW declared this define. :-/ But I'm going to test it. MHD uses a _WIN32: https://gnunet.org/git/libmicrohttpd.git/tree/src/include/platform.h#n97 . I've tested it on MinGW-w64 32 bits and worked fine, but I need to test on MinGW-w64 for 64 bits too. I'm going to test it right now ... :-)

Reply:
Thanks, also, can you run the unit tests (in the /test folder) to check that they all pass as well on Windows?

Reply:
You need libcheck for that but it's available on MinGW: https://libcheck.github.io/check/web/install.html#mingwsource

Reply:
Sure :-) i'm going to do all tests on mingw-w64 32 and 64 and back with feedback :-) please wait it because i think we need to apply some changes on previous patches to yder/ulfius too hehe

Reply:
There's no rush, thanks a lot! :)

Reply:
Awesome! :-)

Finally I've tested it. Please see my test:

```c
#include <stdio.h>
int main() {
#ifdef _MINGW
  printf("Aww yeah, _MINGW is defined here!\n");
#endif
#ifdef _WIN32
  printf("Aww yeah, _WIN32 is defined here!\n");
#endif
  return 0;
}
```

So, I've compiled in on MinGW-w64 for 32 bits:

```bash
duall@duall-dev MINGW32 ~
$ gcc main.c -o test && ./test
Aww yeah, _WIN32 is defined here!
```

and for 64 bits:

```bash
duall@duall-dev MINGW64 ~
$ gcc main.c -o test && ./test
Aww yeah, _WIN32 is defined here!
```

then we need to use `_WIN32`! :-)

Now I'm going to run all tests ...

Reply:
> So, I've compiled in on MinGW-w64 for 32 bits:
> duall@duall-dev MINGW32 ~
> $ gcc main.c -o test && ./test
> Aww yeah, _WIN32 is defined here!

Did you mean `Aww yeah, _MINGW is defined here!` ?

Reply:
Only `_WIN32` is present in both MinGW-w64 32 and 64. There is no `_MINGW` there.

Reply:
Ok, I understand